mido/midifiles/midifiles.py

```diff
@@ -70,6 +88,37 @@ def _dbg(text=''):
     print(text)
 
 
+def _handle_error(error_msg, error_mode):
+    if error_mode == 'raise':
+        raise MidiFileError(error_msg)
+    elif error_mode == 'warn':
+        warnings.warn(error_msg)
+    elif error_mode == 'debug':
+        _dbg(error_msg)
+    elif error_mode == 'ignore':
+        pass
+    else:
+        raise ValueError('unknown error mode {!r}'.format(error_mode))
+
+
+def _clip_data_bytes(data_bytes, error_mode):
```

Comment:
Nice! It's silly that this is the convention among MIDI software (Ardour, Logic, Timidity, etc.), but best be consistent.